### ISSUE TITLE
Digital Climate Strike

### ISSUE THREAD

**Issue by xdamman** [opencollective/opencollective#2360]
Can we add a banner on opencollective.com for the global strike for climate September 20th?
See examples on ‪https://digital.globalclimatestrike.net

And here is the code with customization details: https://github.com/fightforthefuture/digital-climate-strike

This means that our website will be unavailable for one day on Friday September 20th.

At the very least, I’d shutdown our homepage that day.
One could argue that we shouldn’t impact the collective pages. But then, I’d at least put a banner at the top to remind people that today is the global strike for climate.

This is obviously a time sensitive issue. If we decide to do it, we should do it asap (but it shouldn’t take much integration time).

**Comment by xdamman** [comment]
Based on the team conversation this morning. Next steps are:
- [ ] create a draft email to send to all collective admins 
- [ ] Create draft blog post to explain why we are doing this
- [ ] Define behavior of the banner as we don’t want to use off the shelf script that includes a google analytics code.  

The agreement is that we will put the banner throughout the website but only green out the homepage on September 20th. We will also give the option to collectives to opt out of the banner.

**Comment by xdamman** [comment]
Behavior:
- Banner at the top or at the bottom that overlays. 
- Cross button to dismiss it (and save that in localStorage or cookie till end of September)
- Full page overlay on September 20th on the homepage
- Remove both overlay and banner on September 21th

We could use the same design than the one suggested on https://digital.globalclimatestrike.net/ for both the banner and full page overlay.

The only thing I’d change is to add a link to our (future) blog post that explains why we are doing this.

I looked a bit at the code on https://github.com/fightforthefuture/digital-climate-strike
They do offer the option to not include google analytics.

So it feels like the easiest way would be to just use that script (that we can serve ourselves on our own domain) and configure differently for the homepage than for the collective page.

What about we try this to keep things simple, deploy on staging and see how it turns out?
@znarf I know you wanted to implement something ourselves, what was your concern? (My concern was Google Analytics and loading from another domain, but looks like we can solve those two issues)

**Comment by znarf** [comment]
We're a Single Page App, usually dropping a widget like that is really messy. Feel free to experiment and share the result.

But even if it's working on one page, you will figure quickly it's difficult to activate on one page and not on others. How to implement the opt-in/opt-out?

I'm open to ideas, but the safest path is to implement it in React and fully control the behavior.

**Comment by xdamman** [comment]
Ah yes, that's a good point. Let me ask if we can get someone to make a React component for it.

**Comment by xdamman** [comment]
Update: https://github.com/fightforthefuture/digital-climate-strike/issues/55#issuecomment-526942573

TLDR: would be good if one of us could make a react component for it. It might even be helpful to others. 

Can we prioritize this this week? Is there anything else needed to get started on this?

**Comment by xdamman** [comment]
For the record, here is the tweet we sent: https://twitter.com/opencollect/status/1171102933632585728
and this one: https://twitter.com/opencollect/status/1174935068860567552

16 collectives opted in to participate:

- wwcodenyc
- visjs
- korerowellington
- debates
- hledger
- cljdoc
- streetlives
- javers
- dokku
- kitspace
- pytest
- terser
- ghost
- adnauseam
- asciidoctor
- Idyll

Thank you for making this happen!
